delfin/drivers/dell_emc/vmax/rest.py

```diff
@@ -519,6 +520,38 @@ def get_volume_list(self, array, version, params):
             pass
         return device_ids
 
+    def post_request(self, target_uri, payload):
+        """Generate  a POST request.
+        :param target_uri: the uri to query from unipshere REST API
+        :param payload: the payload
+        :returns: status_code -- int, message -- string, server response
+        """
+
+        status_code, message = self.request(target_uri, POST,
+                                            request_object=payload)
+        operation = 'Performance query for URL' % {target_uri}
```

Comment:
Function name is confusing to be generic, but this is perf metric specific function. Please rename.

Reply:
post_request is generic, but I changed message 'Performance  query' to 'POST request'

Reply:
Ok